README.md

```diff
@@ -56,20 +56,6 @@ algorithms for signing your credentials. For now:
   a Bitcoin-based or Ethereum-based ledger)
 * You _can_ use RSA keys to sign, if your use case requires it.
 
-#### Private Key Storage
```

Comment:
I'd like to keep this section and the TODO (private key storage is a crucial topic here).

Reply:
@dlongley said to remove it, but that it is important. Not sure where it should go though. if it helps to get this documentation into a release then I have no issue with it being there, but the consensus was that it belonged somewhere else, just no one was quite sure where.